### ISSUE TITLE
groupByDateAndResource doesn't work with agendas when showing only one day

### ISSUE THREAD

**Issue by lupulin** [fullcalendar/fullcalendar-scheduler#368]
**Issue I'm having:**
I'm using scheduler and I'm confused as to why in agenda view, if you use "groupByDateAndResource" it works for agendas that show more than one day. But for agendas that only show 1 day, it just shows the resource but not the date. For an example: https://codepen.io/lupulin/pen/eGxYod

**What I expect:** 
I would like even agendas that show one day to still have the resource AND the date showing like agendas with more than one day.

Let me know what you think.

Thanks,
Chris

**Comment by arshaw** [comment]
I understand. That would be more consistent. But it might also inconvenience ppl who think that top row with the single date is redundant. I guess they could just set `columnHeader: false`.

**Comment by Brandinga** [comment]
Its also inconsistent to standard fullcalendar.
There is a DateHeader on "Day" view.

Is there a way to bring this maybe as an option to the standard version?
Currently i forked youre repo and have to manage updates per hand :( 

Thanks
Christoph

**Comment by marmotz** [comment]
Yes, it's inconsistent with standard fullcalendar and `columnHeader: false` do the job.

I write a little patch to fix this.

```patch
--- fullcalendar-scheduler.js	2018-06-07 17:36:28.382122653 +0200
+++ fullcalendar-scheduler-patched.js	2018-06-08 16:29:16.033380131 +0200
@@ -2132,18 +2132,11 @@
     // Header Rendering
     // ----------------------------------------------------------------------------------------------
     ResourceDayTableMixin.prototype.renderHeadTrHtml = function () {
-        if (this.daysPerRow > 1) {
-            // do two levels
-            if (this.datesAboveResources) {
-                return this.renderHeadDateAndResourceHtml();
-            }
-            else {
-                return this.renderHeadResourceAndDateHtml();
-            }
+        if (this.datesAboveResources) {
+            return this.renderHeadDateAndResourceHtml();
         }
         else {
-            // do one level
-            return this.renderHeadResourceHtml();
+            return this.renderHeadResourceAndDateHtml();
         }
     };
```

I hope one day this will be fixed

**Comment by pmonnerat** [comment]
I think it would be really consistent to display date in column in 1 day view.
Now, we have a redundant date, the header for all view (excepted 1 day view) and the column.
I case of fullcalender (no scheduler), the first row is always display, also in day (name of the day).
